doc/classes/ScrollContainer.xml

```diff
@@ -31,20 +31,26 @@
 			If [code]true[/code], the ScrollContainer will automatically scroll to focused children (including indirect children) to make sure they are fully visible.
 		</member>
 		<member name="rect_clip_content" type="bool" setter="set_clip_contents" getter="is_clipping_contents" override="true" default="true" />
-		<member name="scroll_deadzone" type="int" setter="set_deadzone" getter="get_deadzone" default="0">
-		</member>
 		<member name="scroll_horizontal" type="int" setter="set_h_scroll" getter="get_h_scroll" default="0">
 			The current horizontal scroll value.
 		</member>
-		<member name="scroll_horizontal_enabled" type="bool" setter="set_enable_h_scroll" getter="is_h_scroll_enabled" default="true">
-			If [code]true[/code], enables horizontal scrolling.
-		</member>
 		<member name="scroll_vertical" type="int" setter="set_v_scroll" getter="get_v_scroll" default="0">
 			The current vertical scroll value.
 		</member>
+		<member name="scroll_horizontal_enabled" type="bool" setter="set_enable_h_scroll" getter="is_h_scroll_enabled" default="true">
+			If [code]true[/code], enables horizontal scrolling.
+		</member>
 		<member name="scroll_vertical_enabled" type="bool" setter="set_enable_v_scroll" getter="is_v_scroll_enabled" default="true">
 			If [code]true[/code], enables vertical scrolling.
 		</member>
+		<member name="scroll_horizontal_visible" type="bool" setter="set_h_scroll_visible" getter="is_h_scroll_visible" default="true">
+			If [code]false[/code], hides the horizontal scrollbar.
+		</member>
+		<member name="scroll_vertical_visible" type="bool" setter="set_v_scroll_visible" getter="is_v_scroll_visible" default="true">
+			If [code]false[/code], hides the vertical scrollbar.
+		</member>
+		<member name="scroll_deadzone" type="int" setter="set_deadzone" getter="get_deadzone" default="0">
```

Comment:
Properties are listed alphabetically, did you change this manually? Running `doctool` will undo those changes.

Reply:
Oh, thanks for catching that, yeah I did do it manually. I put them in the order that they appear in the inspector

Reply:
Okay I put them in alphabetical order